### PR TITLE
Create target

### PR DESCRIPTION
**Pull request by flomosq**
# Create Target

### Trello Card #3

- Create target model
- Topic's target association validation
- User model validate associations and validations
- Validate target model validations
- Validate target model associations
- Create target endpoint
- Create target tests
- Use jbuilder for targets
- Use rspec-json_expectations

### CODE REVIEW

File: spec/requests/api/v1/targets/create_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+RSpec.describe 'POST api/v1/targets', type: :request do
+  let(:user) { create(:user) }
+  let(:topic) { create(:topic) }
+  let!(:target) { attributes_for(:target, topic_id: topic.id) }
```
Comment (santir489) [inline spec/requests/api/v1/targets/create_spec.rb]: there is no need for `!` here

File: spec/requests/api/v1/targets/create_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+RSpec.describe 'POST api/v1/targets', type: :request do
```
Comment (santir489) [inline spec/requests/api/v1/targets/create_spec.rb:1]: Maybe we can add more tests, ie:
- Target has the correct user
- Target was saved
- With some missing/incorrect value

File: app/views/api/v1/targets/show.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1 @@
+json.target @target, partial: 'target', as: :target
```
Comment (santir489) [inline app/views/api/v1/targets/show.json.jbuilder:1]: 🚀

**Review by santir489 — COMMENTED** [review]
Nice! Left some minor comments

File: config/locales/en.yml [hunk 4]
```diff
@@ -30,4 +30,9 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  hello: Hello world
```
Comment (brunvez) [inline config/locales/en.yml:33]: ?

**Review by brunvez — APPROVED** [review]

**Review by santir489 — APPROVED** [review]
🚀

**Pull request merged**